.travis.yml

```diff
@@ -216,10 +216,16 @@ script:
   - $make PLATFORM=sprd-sc9860
   - $make PLATFORM=sprd-sc9860 CFG_ARM64_core=y
 
-  # FSL ls1021a
+  # Layerscape ls1021a
   - $make PLATFORM=ls-ls1021atwr
   - $make PLATFORM=ls-ls1021aqds
 
+  # Layerscape ls1043ardb (ARMv8A)
```

Comment:
Minor: the correct (official) spelling is `ARMv8-A`

Reply:
I will fix it.